### PR TITLE
feat(ui): Implement Unified view Resource details panel

### PR DESCRIPTION
**Pull request by bdauria**
## Description

Please include a short resume of the changes and what is the purpose of PR. Any relevant information should be added to help:
* **QA Team** (Quality Assurance) with tests.
* **reviewers** to understand what are the stakes of the pull request.

**Fixes** # (issue)

## Type of change

- [ ] Patch fixing an issue (non-breaking change)
- [x] New functionality (non-breaking change)
- [ ] Breaking change (patch or feature) that might cause side effects breaking part of the Software
- [ ] Updating documentation (missing information, typo...)

## Target serie

- [ ] 2.8.x
- [ ] 18.10.x
- [ ] 19.04.x
- [ ] 19.10.x
- [x] 20.04.x (master)

<h2> How this pull request can be tested ? </h2>

Please describe the **procedure** to verify that the goal of the PR is matched. Provide clear instructions so that it can be **correctly tested**.

Any **relevant details** of the configuration to perform the test should be added.

## Checklist

- [x] I followed the **coding style guidelines** provided by Centreon
- [ ] I have commented my code, especially new **classes**, **functions** or any **legacy code** modified. (***docblock***)
- [ ] I have commented my code, especially **hard-to-understand areas** of the PR.
- [ ] I have made corresponding changes to the **documentation**.
- [ ] I have **rebased** my development branch on the base branch (master, maintenance).

### CODE REVIEW

File: www/front_src/src/Resources/Details/index.tsx [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+
+const useStyles = makeStyles(() => {
+  return {
+    details: {
+      height: '100%',
+      display: 'grid',
+      gridTemplate: 'auto auto 1fr / 1fr',
+    },
+    header: {
+      gridArea: '1 / 1 / 2 / 1',
```
Comment (Thebarda) [inline www/front_src/src/Resources/Details/index.tsx]: It could be useful to make a course about CSS grid

Comment (bdauria) [reply www/front_src/src/Resources/Details/index.tsx]: @Thebarda Yes, we're going to do a PR in the end with all the translations. It's going to help us involving people in the review that are not frontend devs.

File: www/front_src/src/Resources/Details/Body/ExpandableCard.tsx [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+  };
+
+  const Line = (line, index): JSX.Element => (
+    <Typography key={`${line}-${index}`} variant="body2" component="p">
+      {line}
+    </Typography>
+  );
+
+  return (
+    <Card className={classes.card} color="green">
```
Comment (kduret) [inline www/front_src/src/Resources/Details/Body/ExpandableCard.tsx]: No material-ui console warning about color=green ?

File: www/front_src/src/Resources/colors.ts [hunk 3]
```diff
@@ -0,0 +1,19 @@
+import { lime, purple } from '@material-ui/core/colors';
+
+const downtimeColor = purple[500];
```
Comment (kduret) [inline www/front_src/src/Resources/colors.ts]: I'm currently adding chose colors in the theme in another branch. So we'll update this once PR is merged

File: www/front_src/src/Resources/Chip/Acknowledge.tsx [hunk 4]
```diff
@@ -0,0 +1,13 @@
+import * as React from 'react';
+
+import IconAcknwoledge from '@material-ui/icons/Person';
```
[the remark this answers is not in the document]
Comment (bdauria) [reply www/front_src/src/Resources/Chip/Acknowledge.tsx]: one day I'll get this right :)

**Pull request merged**